src/atoms/ResponsibilityBox.jsx

```diff
@@ -57,6 +60,10 @@ export default function ResponsibilityBox({ responsibility }) {
 }
 
 ResponsibilityBox.propTypes = {
-  responsibility: PropTypes.oneOf(["Inherited", "Hybrid", "Allocated"])
-    .isRequired,
+  responsibilityForControl: PropTypes.oneOf([
+    "Inherited",
+    "Hybrid",
+    "Allocated",
+    null,
```

Comment:
Nice. We should probably mirror this on the backend too. Don't think we have an enum defined anywhere for allowed "responsibilities"